src/nvim/testdir/Makefile

```diff
@@ -7,12 +7,11 @@ ifeq ($(OS),Windows_NT)
 else
   NVIM_PRG ?= ../../../build/bin/nvim
 endif
-TMPDIR ?= Xtest-tmpdir
 SCRIPTSOURCE := ../../../runtime
 
 export SHELL := sh
 export NVIM_PRG := $(NVIM_PRG)
-export TMPDIR
+export TMPDIR := Xtest-tmpdir
```

Comment:
The changes to this file don't do anything other than make it harder to override `TMPDIR`.  What was "wrong" with the old code?

Reply:
On macos TMPDIR is already set to something insane and it makes tests more brittle.

Reply:
Note that our Lua tests don't allow TMPDIR override and no one complained.